pkg/volume/aws_ebs/aws_util.go

```diff
@@ -116,6 +116,11 @@ func verifyDevicePath(devicePaths []string) (string, error) {
 
 // Unmount the global mount path, which should be the only one, and delete it.
 func unmountPDAndRemoveGlobalPath(globalMountPath string, mounter mount.Interface) error {
+	if pathExists, pathErr := pathExists(globalMountPath); pathErr != nil {
+		return fmt.Errorf("Error checking if path exists: %v", pathErr)
+	} else if !pathExists {
```

Comment:
@ciwang @saad-ali Is this expected that we are trying to unmount a path and it does not exist? Are we failing silently?